README.md

```diff
@@ -16,6 +16,13 @@ make -j4
 ctest
 ```
 
+Note, you will need to add the following to your `$PYTHONPATH` in order to run the converters. 
+
+```
+export PYTHONPATH=$PYTHONPATH:/<path_to_ioda-bundle_build>/lib/pyiodaconv
+export PYTHONPATH=$PYTHONPATH:/<path_to_iodaconv_src>/iodaconv/src
```

Comment:
To me the `<path_to_iodaconv_src>` seems to overlap with the `iodaconv` directory. Say you cloned ioda-bundle in your `$HOME/jedi` directory. I read `<path_to_iodaconv_src>` as being the path `$HOME/jedi/ioda-bundle/iodaconv` since that is where the clone of ioda-converters is located.

Perhaps substitute `<path_to_iodaconv_src>` with `<path_to_ioda-bundle_src>` or `<path_to_ioda-bundle>`. 

Thanks!

Reply:
Thanks for fixing this!